Packages/com.unity.inputsystem/Documentation~/SupportedDevices.md

```diff
@@ -29,61 +29,33 @@ The following devices are supported in a way that does not require recognition o
-|Switch|No(6)|No(6)|No|No(6)|No|No|No|No|Yes|No|
+|Xbox 360(4)|Yes|Yes(3)|Yes|Yes|No|No|Yes|No|No|Sometimes(2)|
+|Xbox One|Yes(1)|Yes(3)|Yes(1)|Yes|Yes(1)|No|Yes|No|No|Sometimes(2)|
+|PS4|Yes(5)|Yes(5)|Yes(5)|Yes(5)|Yes(5)|No|No|Yes|No|Sometimes(2)|
+|Switch|Yes|Yes|Yes|Yes|No|No|No|No|Yes|Sometimes(2)|
 
     (1) The trigger motors on the Xbox One controller are only supported on
         UWP and Xbox at the moment.
-    (2) Implementation of the Gamepad W3C spec varies wildly between browsers.
-        We are hoping to improve our support for the mostly wildly used
-        controllers across all major browsers.
+    (2) WebGL support varies wildly between browsers, devices, and OSes.
     (3) XInput controllers on Mac currently require the installation of the
         TattieBogle Xbox 360 controller driver which can be found at
-        https://github.com/360Controller/360Controller. We not yet support
-        consuming raw USB input.
+        https://github.com/360Controller/360Controller. However, the latest
+        generation of Xbox One controllers are natively supported on Macs as HID devices
+        when connected via Bluetooth.
     (4) This includes any XInput-compatible device.
     (5) We do not support the gyro/accelerometer on PS4 controllers on platforms
-        other than the PS4 at the moment.
-    (6) Support for the Switch controller on desktops is being worked on.
+        other than the PS4 at the moment. Also, on such platforms, we only support
+        PS4 controllers when connected via Bluetooth or USB; we do NOT support the
+        "DualShock 4 USB Wireless Adaptor".
 
 ### WebGL
 
->NOTE: There is no rumble support currently on WebGL.
-
->NOTE: WebGL does __NOT__ support running both the old and the new input system in parallel.
-
-The new input system supports the "Standard Gamepad" mapping as specified in the [W3C Gamepad Specification](https://www.w3.org/TR/gamepad/#remapping). We do not yet support gamepads that are surfaced by the browser without a mapping.
-
-The following browser&OS combinations are known to support the "Standard Gamepad" mapping.
-
-|Chrome(Win)|Chrome(Mac)|Chrome(Linux)|Firefox(Win)|Firefox(Mac)|....|
-|-----------|-----------|-------------|------------|------------|----|
-
-### Steam
-
-    NOTE: Steam controller API support is still work-in-progress.
-
-For details about Steam controller API support, please consult the [Steam Support Guide](Steam.md).
+The input system supports the "Standard Gamepad" mapping as specified in the [W3C Gamepad Specification](https://www.w3.org/TR/gamepad/#remapping). We also support gamepads and joysticks that are surfaced by the browser without a mapping, but that support is generally limited to detecting axes and buttons which are present, without any context as to what they mean, which means that this is generally only useful when [manually remapped by the user](HowDoI.md#-create-a-ui-to-rebind-input-in-my-game).
```

Comment:
For the latter part, might be worth clarifying how we surface them. I assume gamepads come out as Joysticks in that case? (like with HID)